pyanaconda/core/kernel.py

```diff
@@ -143,3 +143,6 @@ def getbool(self, arg, default=False):
             elif a == 'no' + arg:
                 result = False  # XXX: should noarg=off -> True?
         return result
+
+
+cmdline = KernelArguments.from_defaults()
```

Comment:
I am not sure if we want to keep the name `cmdline`. I would use `kernel_arguments` instead. Then the imports could be `from pyanaconda.core.kernel import kernel_arguments`. What do others think?

Reply:
Ah, in one word "yes". Solves a lot of baggage.